index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   admission-controller:
   - apiVersion: v2
     appVersion: 0.0.1
-    created: "2023-03-21T20:24:59.716348015Z"
+    created: "2023-04-06T16:31:11.0566198-04:00"
```

Comment:
@hengw0498 there is a difference in time format than before. This might be due to using Windows version of helm? Not sure if this can cause any impact.

Reply:
The first time stamp, "2023-03-21T20:24:59.716348015Z," is in UTC (Coordinated Universal Time) format and was created on March 21, 2023 at 20:24:59.716348015 UTC. The "Z" at the end of the timestamp indicates that it is in UTC format.

The second time stamp, "2023-04-06T16:31:11.0566198-04:00," is in ISO 8601 format and was created on April 6, 2023 at 16:31:11.0566198 in Eastern Daylight Time (EDT), which is four hours behind UTC. The "-04:00" at the end of the timestamp indicates the time zone offset from UTC.

So, the difference between these two time stamps is that they represent times in different time zones.

I can change my local time zone to UTC and push it again but I don't think we really need to do this. It is just a time stamp so that it is fine.

Reply:
Thanks @hengw0498, that sounds good.